urllib3/request.py

```diff
@@ -40,6 +41,12 @@ class RequestMethods(object):
 
     def __init__(self, headers=None):
         self.headers = headers or {}
+        try: 
+            version = sys.modules['urllib3'].__version__
```

Comment:
We should probably just import the `__version__` information in our header instead of inspecting `sys.modules`.

Reply:
I'm not sure if that'll work; we might need to pull `__version__` into a `constants.py`, and import that from both `__init__.py` and `request.py` (or wherever else we decide to put this code).